scripts/import/import_clinvar_xml

```diff
@@ -1458,21 +1509,40 @@ sub clean_old_data{
     $allele_syn_delete_sth->execute() || die "Could not delete entry id $allele_syn_id from allele_synonym\n";
   }
 
+  print "Deleting old variation feature, transcript variation and MTMP transcript variation\n";
+  # remove from variation_feature, transcript_variation and MTMP_transcript_variation
+  my $tv_del_sth = $dba->dbc()->prepare(qq[ select vf.variation_feature_id from variation_feature vf
+                                            left join source s on s.source_id = vf.source_id
+					    where s.name ='ClinVar';
+					    ]);
+  $tv_del_sth->execute() || die "Error selecting variation feature from ClinVar\n";
+  my $tv_to_del = $tv_del_sth->fetchall_arrayref();
+  foreach my $to_del (@{$tv_to_del}){
+    my $vf_id_del = $to_del->[0];
+    my $del_vf_sth = $dba->dbc()->prepare(qq[ delete from variation_feature where variation_feature_id = $vf_id_del ]);
+    my $del_sth = $dba->dbc()->prepare(qq[ delete from transcript_variation where variation_feature_id = $vf_id_del ]);
+    my $del_mtmp_sth = $dba->dbc()->prepare(qq[ delete from MTMP_transcript_variation where variation_feature_id = $vf_id_del ]);
+    $del_vf_sth->execute() || die "Could not delete entry with variation_feature_id = $vf_id_del from variation_feature\n";
+    $del_sth->execute() || die "Could not delete entry with variation_feature_id = $vf_id_del from transcript_variation\n";
+    $del_mtmp_sth->execute() || die "Could not delete entry with variation_feature_id = $vf_id_del from MTMP_transcript_variation\n";
+  }
+
+  print "Deleting old data from other tables\n";
   # remove from other tables
-  my @tables = qw/variation_synonym failed_variation variation_set_variation variation_citation/;
+  my @tables = qw/variation_synonym failed_variation variation_set_variation variation_citation variation_hgvs/;
```

Comment:
The table `variation_hgvs` is updated by the transcript variation pipeline. As we are deleting all the ClinVar entries we also need to delete entries from hgvs otherwise if we run the tv pipeline in update mode some old entries could still be there.